src/environments/environment.staging.ts

```diff
@@ -4,6 +4,6 @@ export const environment = {
     ...prodEnv,
     staging: true,
     production: false,
-    api: 'https://api-staging.civis.vote',
+    api: 'https://civis-api-jpmc-backend.staging-9143.c66.me',
```

Comment:
Instead of updating the api, you'll need to add a new variable like `apiJpmc` and use that variable for your api calls. You'll also won't be required to create another environment file, the existing staging and prod files should suffice.

Reply:
Done